src/themes/mainTheme.ts

```diff
@@ -191,6 +193,9 @@ export default createMuiTheme({
       root: {
         textTransform: 'none',
       },
+      label: {
+        height: '100%',
```

Comment:
for å kunne bruke knapp for trefflisten. Det skal ikke påvirke andre knapper

Reply:
Den er grei